src/csharp/Microsoft.Spark/Sql/SparkSession.cs

```diff
@@ -389,6 +390,41 @@ public void Dispose()
         /// </summary>
         public void Stop() => _jvmObject.Invoke("stop");
 
+        /// <summary>
+        /// Get the <see cref="VersionSensor.VersionInfo"/> for the Microsoft.Spark assembly
+        /// running on the Spark Driver and make a "best effort" attempt in determining the version
+        /// of Microsoft.Spark.Worker assembly on the Spark Executors.
+        /// </summary>
+        /// <param name="numPartitions">Number of partitions</param>
+        /// <returns>
+        /// A <see cref="DataFrame"/> containing the <see cref="VersionSensor.VersionInfo"/>
+        /// </returns>
+        public DataFrame Version(int numPartitions = 10)
```

Comment:
Maybe we should name this `AssemblyVersion` instead just in case Spark ever adds a method called `Version` ?